cliquet/tests/test_permission.py

```diff
@@ -245,6 +245,7 @@ def __init__(self, *args, **kwargs):
                 side_effect=redis.RedisError)]
 
 
+@unittest.skipIf(psycopg2 is None, "postgresql is not installed.")
```

Comment:
Since this is used more than once, we could define it (like ``@skip_if_travis``)